src/lib/components/Cytoscape.react.js

```diff
@@ -818,7 +818,8 @@ Cytoscape.defaultProps = {
     autoRefreshLayout: true,
     generateImage: {},
     imageData: null,
-    responsive: false
+    responsive: false,
+    elements: {}
```

Comment:
Not `[]`?

Reply:
> Both {} and [] are fine since 0.1, so I don't think it matters a lot.

Not according to the `propType`:

https://github.com/plotly/dash-cytoscape/blob/4cba5bcc8aa8110b5365c0995e177b318e09c18d/src/lib/components/Cytoscape.react.js#L534

So either we should change this to something like
```js
elements: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.object), PropTypes.object])
```
Or we should change the default to `[]`

Reply:
Good catch. I'll update the proptypes.